tests/Traits/DataHelpers/ConfigHelper.php

```diff
@@ -56,17 +61,48 @@ private function getDefaultFindologicConfigServiceMock(
         $configServiceMock->method('get')
             ->willReturnMap(
                 [
-                    ['FinSearch.config.active', $salesChannelId, $config['active']],
-                    ['FinSearch.config.shopkey', $salesChannelId, $config['shopkey']],
-                    ['FinSearch.config.activeOnCategoryPages', $salesChannelId, $config['activeOnCategoryPages']],
-                    ['FinSearch.config.crossSellingCategories', $salesChannelId, $config['crossSellingCategories']],
-                    ['FinSearch.config.searchResultContainer', $salesChannelId, $config['searchResultContainer']],
+                    [
```

Comment:
This array is just bloated with new lines. Please revert this.

Reply:
It extends the column count otherwise lint will fail. This is PSR-12. Please decide which coding standard to follow.

Reply:
This is perfect PSR-12 standard. As stated [here](https://www.php-fig.org/psr/psr-12/#47-method-and-function-calls):

> Argument lists MAY be split across multiple lines, where each subsequent line is indented once. When doing so, the first item in the list MUST be on the next line, and there MUST be only one argument per line. A **single argument being split across multiple lines** (as might be the case with an anonymous function **or array**) does **not constitute splitting the argument list** itself.

Reply:
So how come the Phpstorm changes this even though PSR-12 is set as the code style?

Reply:
It might have not complained before as there was no language ID being passed. After passing language ID, it exceeds the column length

Reply:
I think this was a bit of misunderstanding. Yes the project applies PSR-12 and yes, the maximum character limit per lines is 120. This was actually my bad as I thought you added one additional case and now you have moved all lines into this bloated structure, but it actually needs to be like that, since the language id is now also passed.
You do not have to fix anything here, sorry for the confusion ^^.